state/machine.go

```diff
@@ -279,6 +279,14 @@ func (m *Machine) HardwareCharacteristics() (*instance.HardwareCharacteristics,
 	return hardwareCharacteristics(instData), nil
 }
 
+func (m *Machine) DisplayName() (string, error) {
```

Comment:
We don't want to add this exported method just for tests. Someone will end up using it in production code and we've got a double access for machine then instance data.

Instead, add a public method, `GetInstanceData` to _package_test.go_ that wraps the private method `getInstanceData` and call that to get it directly.

Reply:
Ah ok that makes sense, I'll make the change :+1: thanks for the review!

Reply:
I just realized we already have a method called `InstanceNames` in the machine package that I didn't know before, so I just used that one in the tests instead of redundantly adding a new method. Thanks for the catch!